js-exercises/user-check-password/user.js

```diff
@@ -1,17 +1,19 @@
-
 // (*)
 // Follow the instructions and fill in the blank sections.
-function User() {
+function User(username, password) {
   // set a username and password property on the user object that is created
+  this.username = username;
+  this.password = password;
 }
 
 // create a method on User called `checkPassword`
 // this method should take in a string and compare it to the object's password property
 // return `true` if they match, otherwise return `false`
-User.prototype.checkPassword = function checkPassword() {
-
+User.prototype.checkPassword = function checkPassword(enteredPassword) {
+  if (enteredPassword === this.password) {
```

Comment:
`return enteredPassword === this.password` would be enough as this will return a boolean

Reply:
Thanks for the catch, will update the code with correction.

Reply:
Worked on the suggestion and updated the PR accordingly with this commit 6aec16e.